README.md

```diff
@@ -61,6 +61,12 @@ The official constitution of the UNSW Security Society.
       3.6.2 Cease to be a member of the club;
       3.6.3 Cease to be a UNSW student;
       3.6.4 Are absent from any three (3) consecutive meetings of the club without apology or leave; or
+        3.6.4.1 Unless exempted unanimously by the Executive.
```

Comment:
Does this include the executive themselves that may be the target of a removal? If not, should this be explicit?